dracut/99kdumpbase/kdump.sh

```diff
@@ -461,13 +467,16 @@ save_opalcore_ssh() {
 
     dinfo "saving opalcore:$OPALCORE to $3:$1"
 
+    # shellcheck disable=SC2086 # $2 (_ssh_opts) needs to be split
     if ! scp "$2" "$OPALCORE" "$4:$1/opalcore-incomplete"; then
```

Comment:
Hi @coiby,
thanks for taking care of this. I believe you missed the quotes on the line above. Other than that the commit looks fine. Even when having all the checks disabled isn't really ideal.
BTW, when I looked at the problem yesterday I got a little bit side tracked and rewrote `dump_ssh` to use `curl` rather than `ssh/scp`. So far it seems to work all fine and it  cleaned up the code quite a bit. Do you think it makes sense to go that direction?

Reply:
Hi @prudo1,

Oh yes, I missed the quotes. It's fixed in new version, thanks! 

If there is a more elegant solution, that couldn't be better. Can you send a draft patch? I'm also curious to know curl could provide a different solution.